distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -43,7 +43,7 @@ type Array
          Get the last element of an array.
 
              [1, 2, 3].to_array.at -1 == 3
-    at : Integer -> Any ! Index_Out_Of_Bounds
+    at : Integer | Text -> Any ! Index_Out_Of_Bounds
```

Comment:
Why is a `Text` allowed as an array index?

Reply:
Just an left over: [418d8f5](https://github.com/enso-org/enso/pull/7716/commits/418d8f539845f2cdb0b540d45db38365e4721a50)